wallet/wallet.go

```diff
@@ -3706,6 +3713,79 @@ func (w *Wallet) LockedOutpoints() []dcrdtypes.TransactionInput {
 	return locked
 }
 
+func (w *Wallet) acctLockedOutpoints(ctx context.Context, acctName string) ([]dcrdtypes.TransactionInput, error) {
+	w.lockedOutpointMu.Lock()
+	locked := make([]wire.OutPoint, len(w.lockedOutpoints))
+	i := 0
+	for op := range w.lockedOutpoints {
+		locked[i] = op
+		i++
+	}
+	w.lockedOutpointMu.Unlock()
+
+	acctLocked := make([]dcrdtypes.TransactionInput, 0, len(w.lockedOutpoints))
```

Comment:
`len(w.lockedOutpoints)` here while the mutex is unlocked.  Mean to do `len(locked)` instead?